src/modules/Lessons/8_LessonUserData/components/UserForm.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import TextInput from '../../../common/components/TextInput';
+import RadioButtonField from './RadioButtonField';
+import CheckBoxField from './CheckBoxField';
+import Dropdown from '../../../common/components/Dropdown';
+import Saving from './Saving';
+import Countries from '../../../../util/countries.js'
+
+class UserForm extends Component{
+	constructor(props){
+		super(props);
+
+		this.state = {
+			name: '',
+			gender: '',
+			education: '',
+			country: Countries[0].label,
+			isSaving: false
+		}
+		this.onChange = this.onChange.bind(this);
+		this.onSelectChange = this.onSelectChange.bind(this);
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+	onChange(event){
+		console.log('...onChange...');
+		this.setState({
+			[event.target.name]: [event.target.value]
+		});
+	}
+
+	onSelectChange(event){
+		console.log('...onSelectChange...');
+		var index = event.nativeEvent.target.selectedIndex;
+		console.log(event.target.name+' = '+event.nativeEvent.target[index].text);
+
+		this.setState({
+			[event.target.name]: event.nativeEvent.target[index].text
```

Comment:
why do you need to use `nativeEvent`? Just `event.target.value` should be enough for our usecase.